### PR TITLE
Fix: various tab audio handler bugs

### LINKED ISSUE #63: Tab Audio Behavior is inconsistent in Opera
Most of the time, the Tab Audio Behavior does not react when audio starts or stops playing in a tab. instead reacting when the settings tab is selected. Sometimes it may react as intended, however it is uncommon.

### PR DESCRIPTION
**Pull request by PikaDude**
This PR includes:
- Fixed a bug causing the Tab Audio Handler to not work as intended on some browsers. (Closes #63)
- Fixed a bug causing the Tab Audio Handler to get stuck in the wrong state if tabs stopped playing audio while the music was paused and vice-versa.

### CODE REVIEW

File: scripts/background/TabAudioHandler.js [hunk 1]
```diff
@@ -25,6 +27,7 @@ function TabAudioHandler() {
             tabUpdatedHandler = checkTabs;
             chrome.tabs.onUpdated.addListener(checkTabs);
             chrome.tabs.onRemoved.addListener(checkTabs); // A tab that is audible can be closed and will not trigger the updated event.
+            checkTabsInterval = setInterval(checkTabs, 100);
```
Comment (PikaDude) [inline scripts/background/TabAudioHandler.js:30]: This is used as a fallback for some browsers, including some scenarios in Chrome.

**Review by paladique — APPROVED** [review]
👍  edit: just noticed the debug flag is on

File: scripts/background/Utility.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 'use strict';
 
-var DEBUG_FLAG = false;
+var DEBUG_FLAG = true;
```
Comment (paladique) [inline scripts/background/Utility.js]: Looks like this flag may need to be set back to `false`

Comment (jordigarcl) [reply scripts/background/Utility.js]: Reverted

**Comment by jordigarcl** [comment]
Originally did a zipcode link change here but will include it in a separate PR

**Review by jordigarcl — APPROVED** [review]
LGTM, quickly tested the mentioned scenarios in Opera. Could reproduce the bugs in `develop`, couldn't on this PR. 👍

**Pull request merged**